### DISCUSSION TITLE
Can I clock hours for entire month in a single entry?

### DISCUSSION THREAD

**Discussion by surenganne** [kimai/kimai#4009] [General]
I would like to add time for employees for the whole month in one entry. For example, For the month of April, If an employee had spent 76 hours from 1st to 30th April, I would like to have that as one entry.

We are using time tracking on Jira, I don't wish to duplicate the effort. I see this option on one of the versions. I have installed 2.0.16. How can I do a single entry for the entire month?

Can't we have from & to date to enter only duration like in previous versions?

**Comment by kevinpapst** [comment]
You can add a time entry with the duration of 76 hours.

You need to allow that in System > Settings > Timesheet > Maximum duration of a timesheet record in minutes before saving is rejected (0 = deactivated).

But that will heavily conflict with reporting, dashboard widgets and other advanced features that are coming soon.

**Reply by kevinpapst** [reply]
Please open separate issues for separate questions.

And please have a look at the System > Settings and through the Documentation. Time-tracking mode is probably what you are looking for.
